app/main.py

```diff
@@ -1 +1,13 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    action, source_file, destination = command.split()
+    if action == "mv" and "/" in destination:
```

Comment:
"/" in destination
Cheks for / is so bad.
Use os.path.split for check directories